content/blog/multitenancy-for-kubernetes-clusters-using-hpe-alletra-6000-and-nimble-storage.md

```diff
@@ -0,0 +1,234 @@
+---
+title: Multitenancy for Kubernetes clusters using HPE Alletra 6000 and Nimble Storage
+date: 2021-06-15T15:00:00.000Z
+author: Michael Mattsson
+authorimage: /img/portrait-2020-192px.png
+tags:
+  - hpe-nimble-storage
+  - hpe-3par-and-primera
+---
+In a storage infrastructure economy where IT is under constant pressure to deliver more with less, yet still provide a high standard in data services directly to end users without compromising system or data security.
```

Comment:
Incomplete sentence. Please change to something like "We live in a storage infrastructure economy, where IT is under constant pressure to deliver more with less, yet still provide a high standard in data services directly to end users without compromising system or data security."